requirements-transitive.txt

```diff
@@ -0,0 +1,85 @@
+advocate==1.0.0
```

Comment:
Would we ever consider using something like [poetry](https://python-poetry.org/)?

Reply:
Yes! Absolutely. We may have more flexibility out here, but for internal services things like our package vulnerability scanner don't support non-requirements.txt based systems right now :(